reports/hemang.md

```diff
@@ -46,7 +46,7 @@ activity:
  - 2:
    - repo: coala Projects
    - link: https://github.com/coala/projects/commits/master?author=hemangsk
-   - details: > 
+   - details: >
```

Comment:
Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.